.github/workflows/continuous-integration.yml

```diff
@@ -10,13 +10,16 @@ jobs:
         os:
           - ubuntu-latest
           - macos-latest
+          - windows-latest
         java-compiler:
           - default
           - ecj
         exclude:
           # testing ECJ compilation on any one OS is sufficient
           - os: macos-latest
             java-compiler: ecj
+          - os: windows-latest
+            java-compiler: ecj
```

Comment:
Why ECJ on Windows?

Reply:
Actually, why also ECJ on Mac?

Reply:
Take a closer look at the comment and `exclude:` lines just above. Testing ECJ compilation on any one OS is sufficient, so we let ECJ on Ubuntu through and exclude the others.